.github/workflows/pypi_build.yml

```diff
@@ -0,0 +1,50 @@
+# YAML schema for GitHub Actions:
+# https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions
+#
+# Helpful YAML parser to clarify YAML syntax:
+# https://yaml-online-parser.appspot.com/
+#
+
+name: PyPI Build
+
+on:
+  schedule:
+    # 1pm UTC is 6am or 7am PT depending on daylight savings.
+    - cron: '0 13 * * *'
+
+  workflow_dispatch:
+    inputs:
+      upload-type:
+        description: 'Upload type'
+        required: true
+        default: 'test-pypi'
+        type: choice
+        options:
```

Comment:
Can we add an option for `'pypi'` and have that be the default? It would need to drop the `--test-pypi` flag as well.

I think we'd need to use a different TWINE_PASSWORD as well.